packages/api/amplify_api_dart/lib/src/graphql/providers/oidc_function_api_auth_provider.dart

```diff
@@ -20,7 +20,7 @@ class OidcFunctionAuthProvider extends TokenAmplifyAuthProvider {
   Future<String> getLatestAuthToken() async {
     final token = await authProvider.getLatestAuthToken();
     if (token == null) {
-      throw const ApiException(
+      throw const UnknownException(
```

Comment:
I would make this an `ApiOperationException` or you could introduce an `ApiAuthException` for these